indexer/src/main/resources/application.properties

```diff
@@ -22,20 +22,23 @@ indexer.xref.file.path=ftp://ftp.uniprot.org/pub/databases/uniprot/knowledgebase
 proteome.indexing.xml.file=/Users/jluo/projects/github/uniprot-indexer-parent/indexer/testdata/proteome_ds.xml.gz
 # hum disease file
 indexer.disease.file.path=ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/humdisease.txt
+
 ## ==================== uniprotkb indexing properties ====================
-uniprotkb.indexing.chunkSize=100
+uniprotkb.indexing.chunkSize=1000
+uniprotkb.indexing.uniProtKBLogRateInterval=100000
+uniprotkb.indexing.suggestionLogRateInterval=10000
 uniprotkb.indexing.writeRetryLimit=3
 uniprotkb.indexing.writeRetryBackOffFromMillis=50
 uniprotkb.indexing.writeRetryBackOffToMillis=1000
-uniprotkb.indexing.entryIteratorThreads=2
-uniprotkb.indexing.entryIteratorQueueSize=50
+uniprotkb.indexing.entryIteratorThreads=4
+uniprotkb.indexing.entryIteratorQueueSize=100
 uniprotkb.indexing.entryIteratorFFQueueSize=5000
-uniprotkb.indexing.subcellularLocationFile=target/test-classes/uniprotkb/subcell.txt
-uniprotkb.indexing.accessionGoPubmedFile=target/test-classes/uniprotkb/PMID.GO.dr_ext.txt
-uniprotkb.indexing.goDir=target/test-classes/uniprotkb
-uniprotkb.indexing.uniProtEntryFile=target/test-classes/uniprotkb/test-uniprotkb-entries.dat
-uniprotkb.indexing.diseaseFile=target/test-classes/uniprotkb/humdisease.txt
-uniprotkb.indexing.keywordFile=target/test-classes/uniprotkb/keywlist.txt
-uniprotkb.indexing.pathwayFile=target/test-classes/uniprotkb/unipathway.txt
-uniprotkb.indexing.taxonomyFile=/Users/jluo/projects/github/uniprot-indexer-parent/indexer/testdata/public_node_new.dat
-
+uniprotkb.indexing.subcellularLocationFile=/home/eddturner/working/solrcloud/ffs/subcell.txt
```

Comment:
What should be the default path for all the files?

Reply:
good point! i've reverted my accidental changes to what it was before.